assessments/guide/security-reviewer.md

```diff
@@ -43,4 +43,31 @@ expected to conduct an in-depth review described in the [Security Assessment
 Guide](./). Security Reviewers should seek the approval of the other
 participating Security Reviewers and at least 1 co-chair before merging.
 
+## Conflict of interest
 
+There is a possibility of conflict of interest that can arise between a security reviewer and
+the project being reviewed due to the closely-knit nature of the community. Having clear
+guidelines for conflict of interest situations are important to prevent:
+
+- Individuals from intentionally or unintentionally promote their own company's project
+- Sig-security chairs and assessment leads could intentionally or unintentionally limit the participation of an individual unfairly by asserting conflict of interest
+- Security reviews being stalled while groups discuss whether this person or that person should be allowed to participate
+
+The conflicts of interest lie on a spectrum, and are classified into hard and soft conflicts.
+Hard conflicts makes a reviewer ineligible to review a project.
+A soft conflict allows a reviewer to review a project, but not as a [project lead](./project-lead.md).
+A reviewer with a soft conflict is required to document the nature of the conflict in the
+security assessment documentation. It is the responsibility of the security reviewer
+to make known his/her own conflict of interests.
+
+Below are classifications of hard/soft conflicts.
+
+Hard conflicts:
+- Reviewer is a maintainer of the project
+- Reviewer is a direct report of/to a maintainer of the project
+- Reviewer is paid to work on the project
+
+Soft conflicts:
+- Reviewer belongs to the same company/organization of the project, but does not work on the project
+- Reviewer uses the project in his/her work
+- Reviewer has a personal stake in the project (investments, personal relationships, etc.)
```

Comment:
Investments seem pretty serious to me.  I can understand personal friendship with maintainers, but financial benefit seems like a bigger motivation for abuse.

Reply:
I 2nd this. If the reviewer has financial interest in a project that is a hard conflict.

Reply:
Yea i think i agree with the financial interest, but having some trouble coming up with proper phrasing for it...

I'm thinking about the case where someone owns a small amount of stock in a large company or an index. i.e. someone invests in S&P index, which encompass most of the bigger tech firms... I think investment in enterpriseDB vs a review of PostgresDB would probably be a hard conflict but investing in S&P shouldn't be a gatekeeper to someone reviewing keycloak.

Reply:
Financial interest is strange because this doesn't apply directly to open source projects, and at its extreme, I wouldn't have thought that having stock in Google represents conflict of with review of Kubernetes.

Reply:
I think the change @lumjjb added addresses this is nicely in simple language